data/performance.json

```diff
@@ -282,7 +283,8 @@
     "Police Service of Northern Ireland": 1,
     "Police and Crime Commissioner for Bedfordshire": 1,
     "Police and Crime Commissioner for Bedfordshire ": 2,
-    "Police and Crime Commissioner for Cambridgeshire Police": 2,
+    "Police and Crime Commissioner for Cambridgeshire": 1,
+    "Police and Crime Commissioner for Cambridgeshire Police": 1,
```

Comment:
Do we need to consolidate this with the one on line 286 `Police and Crime Commissioner for Cambridgeshire`?

Reply:
I’m not sure. I don’t know if there’s a difference. There official title is the Police and Crime Commissioner for Cambridgeshire and Peterborough according to https://www.cambridgeshire-pcc.gov.uk/about-us/your-commissioner/. I note we also have two different variants of the Police and Crime Commissioner for Bedfordshire (one with a trailing space).

Reply:
Looks like they have updated one service to `Police and Crime Commissioner for Cambridgeshire` but didn't update another one which has `Police` suffix. This can be updated in Toolbox on the service page or service can update it.